packages/syntax-core/src/RichSelect/RichSelectList.tsx

```diff
@@ -59,7 +60,9 @@ export type RichSelectListProps = Omit<
   /** Text shown below select box */
   helperText?: string;
   /** Text shown above select box */
-  label: string;
+  label?: string;
+  //** */
```

Comment:
@jliotta instead of adding new `accessibilityLabel` prop, can we allow for an `id` to be passed in? This follows the same API as `TextField`:

https://github.com/Cambly/syntax/blob/09370d918f3fbb34cab9307ab318e68e03e874f2/packages/syntax-core/src/TextField/TextField.tsx#L106

The way folks can then add a label, is by adding a custom label outside of RichSelectList: https://github.com/Cambly/syntax/blob/09370d918f3fbb34cab9307ab318e68e03e874f2/packages/syntax-core/src/TextField/TextField.tsx#L91-L92